.github/CODEOWNERS

```diff
@@ -51,3 +51,6 @@ privacy-snapshot.json                @MetaMask/extension-privacy-reviewers
 # For now, restricting approvals inside the .devcontainer folder to devs
 # who were involved with the Codespaces project.
 .devcontainer/                       @MetaMask/library-admins @HowardBraham @plasmacorral @brad-decker
+
+# Confirmations UX team to own code for confirmations on UI.
+ui/pages/confirmations               @MetaMask/confirmations-ux
```

Comment:
Can we include the @MetaMask/confirmations-system-team here also?

Wwe maintain the generic `ConfirmationPage` as part of the approval framework, plus we're currently working on alerts and simulations that will have entries within here.

Reply:
I can approve once this change is made

Reply:
I made the update, but found this issue for both the teams: ux and system. We need to fix think before we merge:

![Uploading Screenshot 2024-03-22 at 9.39.02 PM.png…]()

Reply:
Resolved, @jpuri - you all have write access via other methods but the team needs write access in order to be listed as codeowners, which you now have.

Reply:
Thanks @brad-decker